CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased] - 2020-10-03
+### Modified publish_to_test_pypi.yml
+- Skip "Publish to Test PyPi" workflow when there is no change on setup.py
+
 ## [Unreleased]
 ### Modified
 - Add fixture for git user configuration
```

Comment:
@Mastersam07 you should add an entry here, under this line.

Reply:
I don't understand. Check the new changes I pushed if that's what you are referring to